### PR TITLE
EPMDJ-2105. Fixed a change Header mapping setting

### PR DESCRIPTION
**Pull request by ViktoriiaLebedeva**

### CODE REVIEW

File: core/src/main/kotlin/com/epam/drill/endpoints/AgentManager.kt [hunk 1]
```diff
@@ -6,6 +6,7 @@ import com.epam.drill.agentmanager.*
 import com.epam.drill.api.*
 import com.epam.drill.common.*
 import com.epam.drill.endpoints.agent.*
+import com.epam.drill.endpoints.openapi.*
```
Comment (viktorbll) [inline core/src/main/kotlin/com/epam/drill/endpoints/AgentManager.kt]: Why this import?

Comment (ViktoriiaLebedeva) [reply core/src/main/kotlin/com/epam/drill/endpoints/AgentManager.kt]: This is necessary to use SystemSettings, which are a function parameter.
And due to the fact that now we write all the imports through *, I added the import exactly like this

File: core/src/main/kotlin/com/epam/drill/endpoints/openapi/DrillAdminEndpoints.kt [hunk 2]
```diff
@@ -295,10 +295,4 @@ class DrillAdminEndpoints(override val kodein: Kodein) : KodeinAware {
...
     }
-}
-
-@Serializable
-data class SystemSettings(
-    val packagesPrefixes: List<String> = emptyList(),
-    val sessionIdHeaderName: String = ""
-)
\ No newline at end of file
+}
```
Comment (viktorbll) [inline core/src/main/kotlin/com/epam/drill/endpoints/openapi/DrillAdminEndpoints.kt]: newline!

File: core/src/main/kotlin/com/epam/drill/dto/SystemSettings.kt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package com.epam.drill.dto
+
+import kotlinx.serialization.*
+
+@Serializable
+data class SystemSettings(
+    val packagesPrefixes: List<String> = emptyList(),
+    val sessionIdHeaderName: String = ""
+)
```
Comment (viktorbll) [inline core/src/main/kotlin/com/epam/drill/dto/SystemSettings.kt]: newline!

File: core/src/main/kotlin/com/epam/drill/dto/SystemSettings.kt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package com.epam.drill.dto
```
Comment (viktorbll) [inline core/src/main/kotlin/com/epam/drill/dto/SystemSettings.kt]: Who approved this pacakage name?

**Pull request merged**